json_schemer.gemspec

```diff
@@ -26,9 +26,12 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rake", "~> 13.0"
   spec.add_development_dependency "minitest", "~> 5.0"
   spec.add_development_dependency "simplecov", "~> 0.22"
+  spec.add_development_dependency "csv"
   spec.add_development_dependency "i18n"
   spec.add_development_dependency "i18n-debug"
 
+  spec.add_runtime_dependency "base64"
+  spec.add_runtime_dependency "bigdecimal"
```

Comment:
Looks like they don't set versions in Rails either: https://github.com/rails/rails/blob/e3da4fc53d5da7be1d5b93d5659e77c8daddccea/activesupport/activesupport.gemspec#L42-L44